docs/api/api-handler/resolve-context.md

```diff
@@ -0,0 +1,98 @@
+---
+id: resolve-context
+title: ReSolve Context
+description: The resolve context object is available to an API handler function through its request (req) argument. This object implements a communication layer between an API handler and the reSolve framework.
```

Comment:
Do we need to add single quotes to "resolve" (like you did on lines 7 and 9 below)?

Reply:
Added single quotes, thank you.
Below I used backticks, which denote an inline code region in markdown. The 'description' field does not render markdown, so I used single quotes to empasize that 'resolve' is the object's name.

Reply:
Thank you!